### DISCUSSION TITLE
LAPRASユーザーコミュニティへようこそ！

### DISCUSSION THREAD

**Discussion by makikohonjo** [lapras-inc/lapras-user-community#1] [Announcements]
### はじめに
LAPRASはLAPRASユーザーの皆様へより良いサービスや情報提供を行っていくこと、ユーザーの皆様とのコミュニケーションをより充実させていくことを目的とし、GitHubのPublicリポジトリ内で本コミュニティを運営しています。
本コミュニティは[LAPRASユーザーコミュニティ コミュニティガイドライン](https://github.com/lapras-inc/lapras-user-community)に沿って運営されています。

### コミュニティの活用方法
このコミュニティは、現在[LAPRAS](https://lapras.com/)をお使いのユーザーの方を対象としています。
このコミュニティではLAPRASへのご要望や、機能・サービスへのフィードバックなどをお気軽に投稿していただく他、他のLAPRASユーザーの方との交流やディスカッションなどにご活用ください！
LAPRASからは、ユーザーの皆様へのご意見募集や、新しい試みを試験的に公開する場として活用させていただきます。

※ 個別のお問合せや個人情報等を含む内容につきましては[LAPRAS](https://lapras.com/)にログインしていただき、サポートチャットよりお問合せください

### コミュニティへの参加方法
他のコミュニティ参加者との交流や活発な意見交換、LAPRASへのご意見などを歓迎します。
各トピックにご自由にコメント、リアクションしてください。まずは既に立ち上がっているトピックへコメントしてみてください！
新しいトピックを自由に立てることができます。

**早速このトピックにコメント＆自己紹介をどうぞ！**

**Comment by UserSuccessTeamLAPRAS** [comment]
まずはお気軽にコメントを記入してみてください！よろしければ**LAPRASの公開ポートフォリオ**もどうぞ！

**Comment by motuo1201** [comment]
[ドクターメイト株式会社](https://doctormate.co.jp/)でEMをやっています、榎本と申します。
LAPRASの公開スコアはこちら
https://lapras.com/public/motuo

よろしくおねがいします！

**Reply by UserSuccessTeamLAPRAS** [reply]
LAPRASのコミュニティへようこそ！

**Comment by cybist** [comment]
LAPRASのスコア機能いいですよね。
EXP期能も追加されたみたいで、こういうのが数値化されることで、自分のスキルをゲーム感覚で楽しく伸ばしていきたくなります。
平均値は超えておきたい！とGithubやQiitaの更新がんばったりするのは私だけじゃないですよね？

技術者の持つ本来のポテンシャルを楽しく指標化しつつ、採用する会社にとっても必要としているスキルを一定の客観性の下で探すことのできるいいサービスだと思います。

NewsPicksとかSchoo、Youtubeの技術系やビジネス系の投稿とか、もっと広い外部サービスも拾えるようになるといいですねー。

**Reply by UserSuccessTeamLAPRAS** [reply]
cybistさん、LAPRASのコミュニティへようこそ😄
LAPRASを楽しく使っていただいているのが伝わってくるコメント、ありがとうございます！
